oak_containers_stage1/src/main.rs

```diff
@@ -37,7 +37,7 @@ use tokio::process::Command;
 
 #[derive(Parser, Debug)]
 struct Args {
-    #[arg(default_value = "http://10.0.2.2:8080")]
+    #[arg(default_value = "http://10.0.2.100:8080")]
```

Comment:
I wonder if we should not have the default value here, and instead make this an Option and fall back on a value defined as a const somewhere. But not sure that's a good idea, feel free to ignore it.

Reply:
Ignoring it for now.